src/test/java/com/splunk/kafka/connect/SplunkSinkTaskTest.java

```diff
@@ -257,6 +257,7 @@ public void checkExtractedTimestamp() {
         config.put(SplunkSinkConnectorConfig.ENABLE_TIMESTAMP_EXTRACTION_CONF, String.valueOf(true));
         config.put(SplunkSinkConnectorConfig.REGEX_CONF, "\\\"time\\\":\\s*\\\"(?<time>.*?)\"");
         config.put(SplunkSinkConnectorConfig.TIMESTAMP_FORMAT_CONF, "MMM dd yyyy HH:mm:ss.SSS zzz");
+//        config.put(SplunkSinkConnectorConfig.TIMESTAMP_TIMEZONE_CONF, "");
```

Comment:
I think we can remove it, as it defaults to empty string.

Reply:
yes I agree =)